### PR TITLE
uplode anew signinpage

### PR DESCRIPTION
**Pull request by aimanzi**

### CODE REVIEW

File: Pages Files/signin.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            <div>
+                <button id="login" class="btn btn-primary">Log In </button>
+            </div>
+            <div>
+                <button class="btn btn-primary" id="forgetpassword">Forget Your Password ?</button>
+            </div>
+        </div>
+        <div class="signup-contaner">
+            <h3> You Didn't Sign Up Yet ?</h3>
+            <button class="btn btn-primary" id="signup">Sign Up </button>
```
Comment (nevensh) [inline Pages Files/signin.html:38]: I can't see that you are linked the signUp page .
please work on it .

**Review by nevensh — CHANGES_REQUESTED** [review]
please fix it .

File: Pages Files/signin.html (hunk 1 above)
Comment (aimanzi) [reply Pages Files/signin.html:38]: all the linke's between pages well be by DOM elements and eventes on the main javascript file

Comment (nevensh) [reply Pages Files/signin.html:38]: ok aiman , Now i get understand you .

**Review by nevensh — APPROVED** [review]
very good work . 
Continue at this level.

**Pull request merged**